go-controller/pkg/ovn/libovsdbops/acl.go

```diff
@@ -67,6 +77,60 @@ func findACL(nbClient libovsdbclient.Client, acl *nbdb.ACL) error {
 	return nil
 }
 
+// FindRejectACLs looks up the acls with reject action
+func FindRejectACLs(nbClient libovsdbclient.Client) ([]nbdb.ACL, error) {
+	rejectACLLookupFcn := func(acl *nbdb.ACL) bool {
+		return acl.Action == nbdb.ACLActionReject
+	}
+
+	acls, err := findACLsByPredicate(nbClient, rejectACLLookupFcn)
+	if err != nil {
+		return nil, err
+	}
+
+	return acls, nil
+
+}
+
+// FindACLsByPriorityRange looks up the acls with priorities within to a given inclusive range
+func FindACLsByPriorityRange(nbClient libovsdbclient.Client, startPriority, endPriority int) ([]nbdb.ACL, error) {
+	// Lookup all ACLs in the specified priority range
+	priorityRangeLookupFcn := func(item *nbdb.ACL) bool {
+		return (item.Priority <= startPriority ||
+			item.Priority >= endPriority)
+	}
+
+	acls, err := findACLsByPredicate(nbClient, priorityRangeLookupFcn)
+	if err != nil {
```

Comment:
nit: in all these public helper functions you dont need to check err, findACLsByPredicate will return nil, err if there is an error. You can simply do 

```
return findACLsByPredicate(nbClient, priorityRangeLookupFcn)
```